src/main/java/me/bariahq/bariahqmod/Sideboard.java

```diff
@@ -0,0 +1,83 @@
+package me.bariahq.bariahqmod;
+
+import me.bariahq.bariahqmod.config.ConfigEntry;
+import me.bariahq.bariahqmod.shop.ShopData;
+import me.bariahq.bariahqmod.util.FUtil;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.player.PlayerJoinEvent;
+import org.bukkit.event.player.PlayerQuitEvent;
+import org.bukkit.scoreboard.*;
+
+public class Sideboard extends FreedomService {
+
+    public Sideboard(BariaHQMod plugin)
+    {
+        super(plugin);
+    }
+
+    @Override
+    protected void onStart() {
+
+    }
+
+    @Override
+    protected void onStop() {
+
+    }
+
+    ScoreboardManager manager = Bukkit.getScoreboardManager();
```

Comment:
From this line to line 36, please put it under onJoin method. It looks ugly here.